R/rmnorm.R

```diff
@@ -11,13 +11,15 @@
 #' the size of the specified covariance matrix.
 #'
 #' @export
+#' @examples
+#' rmnorm(n = 10, S = covmatrix(sib = 0, hsq = 0.5))
```

Comment:
Excellent that you use `covmatrix()` in the example! 🥜

Reply:
<3